content/v1.11/getting-started/provider-aws.md

```diff
@@ -1156,17 +1156,16 @@ The {{< hover label="providerconfig" line="11">}}spec.credentials.secretRef.name
 A _managed resource_ is anything Crossplane creates and manages outside of the Kubernetes cluster. This creates an AWS S3 bucket with Crossplane. The S3 bucket is a _managed resource_.
 
 {{< hint type="note" >}}
-AWS S3 bucket names must be globally unique. To generate a unique name the example uses a random hash. 
-Any unique name is acceptable.
+AWS S3 bucket names must be globally unique. To generate a unique name the
+example uses `generateName` instead of `name`. Any unique name is acceptable.
 {{< /hint >}}
 
 ```yaml {label="xr"}
-bucket=$(echo "crossplane-bucket-"$(head -n 4096 /dev/urandom | openssl sha1 | tail -c 10))
 cat <<EOF | kubectl apply -f -
 apiVersion: s3.aws.upbound.io/v1beta1
 kind: Bucket
 metadata:
-  name: $bucket
+  generateName: crossplane-bucket-
```

Comment:
What's the suffix to the generated resource? Will it look like the `kubectl get bucket` output below?

Reply:
Also is there an easy/deterministic way to provide a `kubectl delete` command for the generated name?

Reply:
It will look quite similar - I think technically it's always a random 5 character suffix though. I think adding a label would be the best way to deterministically delete. I'll update.